lib/testRun.js

```diff
@@ -63,6 +63,11 @@ module.exports = function() {
       if (text.match(/^[a-z0-9]+$/)) {
         annotation.key = text;
       } else {
+        var active = parseAnnotationValue(text.toLowerCase(), /active\=[\"\']?([1|0|true|false|yes|no])[\"\']?/);
```

Comment:
If you add the `i` flag to the regexp, you don't have to use `toLowerCase()`, which would be more consistent with the rest of the code.